kvrocks.conf

```diff
@@ -174,6 +174,20 @@ slave-read-only yes
 # By default the priority is 100.
 slave-priority 100
 
+# Change the default timeout in milliseconds for socket connect during fullsync.
+# The default value is 3100, and 0 means no timeout.
+#
+# If the master is unreachable before connecting, not having a timeout may block future
+# 'clusterx setnodes' commands because the replication thread is blocked on connect.
+slave-fullsync-connect-timeout 3100
```

Comment:
Nit: we need to denote it's microsec / millsecs or other

Reply:
Yeah we can rename it with a `ms` suffix.

Reply:
> Yeah we can rename it with a `-ms` suffix.

ok